app/about_us/OurCrew.js

```diff
@@ -0,0 +1,26 @@
+import styles from "./OurCrew.module.css";
+import { ourCrewAbout, ourCrewData } from "./crew.js";
+
+export const OurCrew = () => {
+  return (
+    <section className={styles.container}>
+      <header className={styles.crewHeader}>
```

Comment:
This is a small mistake.
The `<header>` element is meant to be a fixed section which is usually on the top of each website.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

In this case this should be a `<div>` or you can just use the `<p>` tag directly.

Reply:
Got it!